internal/github/checkStalePRs.go

```diff
@@ -0,0 +1,68 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/go-github/v60/github"
+
+	"github.com/chia-network/github-bot/internal/config"
+)
+
+// CheckStalePRs will return a list of PR URLs that have not been updated in the last 7 days by internal team members.
+func CheckStalePRs(githubClient *github.Client, internalTeam string, cfg config.CheckStalePending) ([]string, error) {
+	var stalePRUrls []string
+	cutoffDate := time.Now().Add(7 * 24 * time.Hour) // 7 days ago
+	teamMembers, err := GetTeamMemberList(githubClient, internalTeam)
+	if err != nil {
+		return nil, err
+	}
+	communityPRs, err := FindCommunityPRs(cfg.CheckStalePending, teamMembers, githubClient)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, pr := range communityPRs {
+		stale, err := isStale(githubClient, pr, teamMembers, cutoffDate) // Handle both returned values
+		if err != nil {
+			log.Printf("Error checking if PR is stale: %v", err) // Log or handle the error
```

Comment:
This logline should contain the repo owner/name for more context, since this bot will be working with multiple repos.

Reply:
Added the reponame